test/no-hex-escape.js

```diff
@@ -0,0 +1,62 @@
+import test from 'ava';
+import avaRuleTester from 'eslint-ava-rule-tester';
+import rule from '../rules/no-hex-escape';
+
+const ruleTester = avaRuleTester(test, {
+	env: {
+		es6: true
+	}
+});
+
+const error = {
+	ruleId: 'no-hex-escape',
+	message: 'Use unicode escapes over hexadecimal escapes'
+};
+
+ruleTester.run('no-hex-escape', rule, {
+	valid: [
+		`const foo = 'foo'`,
+		`const foo = '\\u00b1'`,
+		`const foo = '\\u00b1\\u00b1'`,
+		`const foo = 'foo\\u00b1'`,
+		`const foo = 'foo\\u00b1foo'`,
+		`const foo = '\\u00b1foo'`
```

Comment:
Can you add tests with numeral values, because they are considered as `Literal` too. At least one like this
```
const foo = 3;
```